### DISCUSSION TITLE
POST issues

### DISCUSSION THREAD

**Discussion by wcrooker** [firefly-iii/firefly-iii#6671] [General]
Hello!

I'm trying to upload the CSV file and JSON file with POST. The curl command I'm using is

`curl --location --request 'http://192.168.50.228:8080/autoupload?secret=MY_SECRET' --header 'Accept: application/json' --header 'Authorization: MY_SECRET' --form 'file=>"/home/apollo/Desktop/Upload/importable.csv"' --form 'json=>"/home/apollo/Desktop/Upload/json.json"'`

But I get an error saying that the json needs to be a file. This is on an UnRaid server also, not sure if that makes a difference.

Thanks!

**Comment by JC5** [comment]
The curl command to include a file is different.

https://docs.firefly-iii.org/data-importer/usage/post/

**Comment by wcrooker** [comment]
Hello,

I tried the documented way, but I always get an error if I attempt to use "json=@" and "importable=@" the error is curl: (26) Failed to open/read local data from file/application. The only way I can get it to do anything is have changing it to file and using "=>".

Thanks!

**Comment by wcrooker** [comment]
Thanks for the quick response. I really appreciate it!

**Comment by wcrooker** [comment]
I installed FireFly and FIDI locally on a VM instead of Docker and it solved this issue. Thanks!
